src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -153,11 +154,6 @@ export function useTaprootInscriptionsInfiniteQuery() {
         calculatedOffset = offset + (total - offset);
       }
 
-      // if we reached the end of the list, start from the beginning
-      if (calculatedOffset >= total) {
-        calculatedOffset = 0;
```

Comment:
here was actual problem, offset was set to 0 and it prevented the script from changing addresses param